wifi_enable.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+echo "This command must be run as SuperUser and using Bash"
+echo "Please give this Pi a unique ID (for the WiFi Access Point):"
```

Comment:
What if we checked to see if the hostname had not been updated, and we requested a new hostname, like:

    if [[ $(hostname) == "localhost" ]]
    then
      echo -n "Name this system: "
      read newhost
      sudo hostname $newhost
    fi